runtime/oti/j9nonbuilder.h

```diff
@@ -2017,6 +2017,7 @@ typedef struct J9BCTranslationData {
 #define BCT_InitInstanceFields  19
 #define BCT_ErrConstantPoolMapTooSmall  14
 #define BCT_InitCopyDoubles  5
+#define BCT_JavaMaxMajorVersionShifted BCT_Java10MajorVersionShifted
```

Comment:
Please flip the order so its 8, 9, 10, Max. Leave a blank line after Max to make it more noticable, and add a comment before 8 to say something like when adding a new major version update BCT_JavaMaxMajorVersionShifted to the maximum allowed value.